conjure-java-client-verifier/src/test/java/com/palantir/conjure/java/compliance/AutoDeserializeTest.java

```diff
@@ -41,7 +42,9 @@
     public static final VerificationServerRule server = new VerificationServerRule();
 
     private static final Logger log = LoggerFactory.getLogger(AutoDeserializeTest.class);
-    private static final AutoDeserializeService testService = VerificationClients.autoDeserializeService(server);
+    private static final AutoDeserializeService jerseyTestService = VerificationClients.autoDeserializeService(server);
+    private static final BlockingAutoDeserializeService dialogueTestService =
```

Comment:
comment to justify that this probably gives us good enough coverage of the async one too?